micromagneticmodel/dynamics/zhangli.py

```diff
@@ -42,10 +62,12 @@ class ZhangLi(DynamicsTerm):
 
         A single scalar value can be passed.
 
-    u : number.Real, discretisedfield.Field
+    u : number.Real, array-like, discretisedfield.Field
 
-        `numbers.Real` can be passed, or alternatively
-        ``discretisedfield.Field`` can be passed.
+        Velocity of the spin current in m/s. If a scalar value or ``Field`` with
```

Comment:
As much as I know, this is know as the "spin-drift velocity". However, it is nitpicking so it is up to you if you want to change or keep it as is.

Reply:
```suggestion
        Spin-drift velocity in m/s. If a scalar value or ``Field`` with
```